### ISSUE TITLE
Maps don't display on iPad Mini 1st gen

### ISSUE THREAD

**Issue by rmohns** [b3ll/Ignition#31]
IPad mini 1st gen., wifi only model, iOS 8.1, Pangu jailbreak, no other crazy tweaks installed.

![image](https://cloud.githubusercontent.com/assets/2131133/5579082/f6b00832-9002-11e4-8555-eb22921f3f5f.jpg)

**Comment by b3ll** [comment]
Part of me wants to think this is because of the older GPU or something... will investigate

**Comment by 1liminal1** [comment]
Hi guys, any update on this. Im having a similar issue where it starts to load and then crashes back to the CarPlay UI. 1st gen mini wifi + cellular

**Comment by b3ll** [comment]
Haven't had time to look into it, will update when I do

**Comment by 1liminal1** [comment]
Fair enough mate, look forward to this fix. And just a thank you for your work on this, great app :)

**Comment by wkw19521** [comment]
No maps icon displaying ipad air

**Comment by 1liminal1** [comment]
Hey, its been a little while, will you be having a look @ this issue, or should I try using another device?
